src/handlebars/support.handlebars

```diff
@@ -476,27 +492,36 @@
       </p>
     </div>
 
-    <div class="anchor" onmouseover="document.getElementById('anchor5').style.visibility='visible'" onmouseout="document.getElementById('anchor5').style.visibility='hidden'"><a href="#support-levels" class="anchor">
-      <img src="dist/assets/anchor.png" id="anchor5" name="anchor" alt="anchor icon"></a>
-      <h2 class="bold"><a id="support-levels">Support Levels</a></h2>
+    <div class="anchor" onmouseover="document.getElementById('anchor6').style.visibility='visible'" onmouseout="document.getElementById('anchor6').style.visibility='hidden'"><a href="#jck" class="anchor">
+      <img src="dist/assets/anchor.png" id="anchor6" name="anchor" alt="anchor icon"></a>
+      <h2 class="bold"><a id="jck">Java Compatibility Kit (JCK) / TCK Compliance</a></h2>
     </div>
     <div class="margin-bottom">
       <p>
-        As a community of open source developers, our commitment is to triage any issues raised and champion them in the appropriate source code project. Of course, if the problem arises from the way we build and test the code we can fix that directly -- but
-        other bugs will be fixed on a "best effort" basis by the correct source project community. If you are looking for higher levels of assurance you should contact commercial companies offering support on these binaries.
-        </br></br>We welcome feedback on any aspect of the project.
+        At this stage the London Jamocha Community CIC (aka LJC) has not been able to reach an
+        agreement with Oracle to use the Java SE Technology Compatibility Kit (TCK)
+        under the terms of the OpenJDK Community TCK License Agreement (OCTLA).</br>
+        </br>
+        We will continue to work with Oracle on this matter.</br>
+        </br>
+        All AdoptOpenJDK binaries are tested with our suite of functional, integration,
+        and performance tests, including real workloads from popular libraries
+        frameworks, languages and applications. We remain confident in the quality
+        of our builds.
```

Comment:
propose alternate wording**: 
"All AdoptOpenJDK binaries are tested with our suite of functional, integration, stress, and performance tests, including real workloads from popular languages and applications. We are confident in the quality of our builds."

** Note: no longer running the commercial/proprietary TCK suite does not raise a concern over quality, compliance does not equate to quality.  

Analogy, restaurant with dress code, employee may be compliant because they wear the right colour of shirt and tie their hair back, but that in no way indicates if they are a good employee that can do a good job during a busy lunch hour.

Reply:
Fixed (mostly)